docs/mllib-feature-extraction.md

```diff
@@ -617,3 +617,43 @@ println("PCA Mean Squared Error = " + MSE_pca)
 {% endhighlight %}
 </div>
 </div>
+
+## Significant Selector
+Idea of this transformation it safe reduce big vector that was produced by Hashing TF for example
+for reduce requirement of memory for manipulation on them. 
+
+This transformation create a model that keep only indices that has different values on fit stage.
```

Comment:
"This transformation creates a model which drops all indices that were constant across all training instances seen in SignificantSelector#fit()"